### PR TITLE
Add org.kde.dolphin

### PR DESCRIPTION
**Pull request by grulja**
Dolphin is a file manager written in Qt and KDE Frameworks.

### CODE REVIEW

File: run_dolphin.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ ! -f "$XDG_CONFIG_HOME/first_start" ]; then
+    echo "First start: running kbuildsycoca5"
+    touch $XDG_CONFIG_HOME/first_start
+    # Dolphin requires that ksycoca cache exists, but cannot run kbuildsycoca5
+    # automatically (because KDED lives outside of the sandbox).
```
Comment (AsciiWolf) [inline run_dolphin.sh]: "KDED" - typo?

**Comment by AsciiWolf** [comment]
bot, build org.kde.dolphin

File: run_dolphin.sh (hunk 1 above)
Comment (grulja) [reply run_dolphin.sh]: Nope, KDED = KDE Deamon.

**Comment by nedrichards** [comment]
@grulja is there anyone else from upstream who'll want access to this repo?

**Comment by grulja** [comment]
Maybe @aleixpol?

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.kde.dolphin

@grulja, @aleixpol, as usually, make sure to accept the invitation to get write access. Thanks!

**Pull request closed without merging**